### PR TITLE
chore(readme): change linking tutorial to pnpm link

### PR DESCRIPTION
**Pull request by TT1228**
# Summary

Change linking tutorial to `pnpm link` since current package management tool is pnpm.

### CODE REVIEW

**Review by Leopoldthecoder — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -265,14 +265,14 @@ In some cases HMR (hot module reloading) is not working out of the box in this c
 1. In the folder `public-ui-components/packages/{workspace}/forms` run
 
     ```sh
-    yarn unlink
+    pnpm remove @kong-ui-public/forms -g
```
Comment (adamdehaven) [inline README.md:268]: I don't think you'd want to remove the dependency; just unlink here, right?

File: README.md [hunk 2]
```diff
@@ -265,14 +265,14 @@ In some cases HMR (hot module reloading) is not working out of the box in this c
...
     ```
 
 1. In the root folder of the host application/package run:
 
     ```sh
-    yarn unlink "@kong-ui-public/forms"
-    yarn install --force --frozen-lockfile
+    pnpm unlink -g @kong-ui-public/forms
+    pnpm install --force --frozen-lockfile
```
Comment (adamdehaven) [inline README.md:275]: You shouldn't need to force the reinstall
```suggestion
    pnpm install
```

File: README.md (hunk 1 above)
Comment (TT1228) [reply README.md:268]: Normally we use `pnpm link -g` to provide module to global `node_modules` folder. If we're removing a global linked module, we need to run `pnpm remove` or `pnpm uninstall`
截屏2024-12-06 下午5 45 07

File: README.md (hunk 2 above)
Comment (TT1228) [reply README.md]: Good point, i've deleted those options.

File: README.md (hunk 1 above)
Comment (adamdehaven) [reply README.md:268]: Ah then make sure the global flag is at the front (not sure it matters, but consistency)

Comment (TT1228) [reply README.md]: OK, i've updated it.

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**